addons/test_mail/tests/test_mail_message.py

```diff
@@ -23,6 +24,41 @@ def setUpClass(cls):
         })
         cls.Message = cls.env['mail.message'].with_user(cls.user_employee)
 
+    @users('employee')
+    def test_message_update_content_bus(self):
+        self.maxDiff = None
+        channel_id = self.env["discuss.channel"].create({
+            "name": "MyTestChannel",
+            "channel_member_ids": [Command.create({"partner_id": self.user_employee.partner_id.id})],
+        })
+        message = self.env['mail.message'].with_user(self.user_employee).create({
```

Comment:
The test is already fully ran as employee user normally. Also better use message post rather than manually using create.